DQMOffline/Trigger/python/PhotonMonitor_cff.py

```diff
@@ -0,0 +1,46 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.PhotonMonitor_cfi import hltPhotonmonitoring
+
+#HLT_SinglePhoton200_IDTight
+SinglePhoton300_monitoring = hltPhotonmonitoring.clone()
+SinglePhoton300_monitoring.FolderName = cms.string('HLT/Photon/Photon300/')
+SinglePhoton300_monitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Photon300_NoHE_v")
+
+
+# HLT_SinglePhoton175_IDTight
+SinglePhoton175_monitoring = hltPhotonmonitoring.clone()
+SinglePhoton175_monitoring.FolderName = cms.string('HLT/Photon/Photon175/')
+SinglePhoton175_monitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Photon175_v*")
+
+
+Photon60_monitoring = hltPhotonmonitoring.clone()
+Photon60_monitoring.FolderName = cms.string('HLT/Photon/Photon60/')
+Photon60_monitoring.denGenericTriggerEventPSet.hltPaths = cms.vstring()
+Photon60_monitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Photon60_R9Id90_CaloIdL_IsoL_v*")
```

Comment:
yes it is available kevin requested them
https://its.cern.ch/jira/browse/CMSHLT-1327